CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+## Contributor Covenant Code of Conduct
```

Comment:
Is this the same as the one in PUP-4? If so, thoughts on linking to it instead of having to maintaining two copies?

Reply:
This is the one from the PUP, but we should link to a standalone Code of Conduct and not link to the PUP as the PUP is worded as a proposal, and though it has been accepted, it is not clear that it was accepted in this form or that this is final and binding.  Is it anywhere else?  I didn't see it in our docs at all.

Reply:
We have considered having the PUPs on our website. They are proposals but also living documents and are meant to be the canonical sources of information. What about linking to the PUP with some verbiage to explain that the proposal and how it contains our official CoC?

Reply:
I think it makes sense to have the CoC as a separate file outside of the PUP. I don't think people reading our code of conduct need to see the proposal, unless they want to.